pkg/common/k8s_utils.go

```diff
@@ -147,3 +156,23 @@ func GetLabel(obj metav1.Object, key string) string {
 	}
 	return obj.GetLabels()[key]
 }
+
+func GenerateClusterID(ctx context.Context, c client.Client) (error, string) {
+	//Already calculated? return it
+	if clusterID != "" {
+		return nil, clusterID
+	}
+
+	ns := &corev1.Namespace{}
+	err := c.Get(ctx, client.ObjectKey{Name: clusterIDNamespace}, ns)
+	if err != nil {
+		return err, ""
+	}
+	clusterID = generateIDFromUID(ns.UID)
+	return nil, clusterID
+}
+
+func generateIDFromUID(uid types.UID) string {
```

Comment:
do we want to use md5 what about the short code work that was used before?

Reply:
I didn't give it much thought, I'll update it to use the existing base36 function.